resources/views/people/_header.blade.php

```diff
@@ -18,7 +18,7 @@
           @if ($contact->birthdate && !($contact->is_dead))
             @if ($contact->birthdate->getAge())
               <span class="{{ htmldir() == 'ltr' ? 'mr1' : 'ml1' }}">@include('partials.icons.header_birthday')</span>
-              <span>{{ $contact->birthdate->getAge() }}</span>
+              <span>{{$contact->birthdate->toShortString()}} ({{ $contact->birthdate->getAge() }})</span>
```

Comment:
Don't use Carbon::toShortString but DateHelper::getShortDate instead:
```suggestion
              <span>{{ \App\Helpers\DateHelper::getShortDate($contact->birthdate) }} ({{ $contact->birthdate->getAge() }})</span>
```

Reply:
My bad, I didn't saw birthdate is a SpecialDate, so it' ok ;)